doc/source/user/basics.broadcasting.rst

```diff
@@ -47,11 +40,23 @@ array([ 2.,  4.,  6.])
 The result is equivalent to the previous example where ``b`` was an array.
 We can think of the scalar ``b`` being *stretched* during the arithmetic
 operation into an array with the same shape as ``a``. The new elements in
-``b`` are simply copies of the original scalar. The stretching analogy is
+``b``, as shown in :ref:`broadcasting.figure-1`, are simply copies of the
+original scalar. The stretching analogy is
 only conceptual.  NumPy is smart enough to use the original scalar value
 without actually making copies so that broadcasting operations are as
 memory and computationally efficient as possible.
 
+.. figure:: broadcasting_1.svg
+    :alt: A scalar is broadcast to match the shape of the 1-d array it
+          is being multiplied to.
+    :name: broadcasting.figure-1
+
+    *Figure 1*
+
+    *In the simplest example of broadcasting, the scalar ``b`` is
+    stretched to become an array of same shape as ``a`` so the shapes
+    are compatible for element-by-element multiplication.*
```

Comment:
nit: The formatting here is a bit off because of the emphasis
```suggestion
    *In the simplest example of broadcasting, the scalar* ``b`` *is
    stretched to become an array of same shape as* ``a`` *so the shapes
    are compatible for element-by-element multiplication.*
```